bitnami/minio/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 3.6.1
+version: 3.6.2
```

Comment:
It is adding a feature, so I think it should bump the minor version

Reply:
Ok will do